cpp/src/arrow/compute/exec/hash_join_benchmark.cc

```diff
@@ -126,6 +129,22 @@ class JoinBenchmark {
 
     join_ = *HashJoinImpl::MakeBasic();
 
+    HashJoinImpl* bloom_filter_pushdown_target = nullptr;
+    std::vector<int> key_input_map;
+
+    bool bloom_filter_does_not_apply_to_join =
```

Comment:
Well we benchmark to see what kind of performance impact the Bloom filter has. 
But since we currently only do early-elimination of rows, and only build on the build side, this disqualifies some types of joins, so we check that here.